tools/health/isAlive

```diff
@@ -85,13 +85,6 @@ def isElkAlive() :
     okString = '"cluster_name"'
     return monitorUtil.outputChecker(cmd, okString, 1)
 
-def isKafkaAlive() :
```

Comment:
I believe this is the start of dumping it. As stated in our discussion I believe `consul` should hold the state of all components consistently, which won't need to rely on `isAlive`. I'll continue the dump of it.